web/src/message_store.ts

```diff
@@ -1,6 +1,7 @@
 import * as blueslip from "./blueslip";
 import * as people from "./people";
 import type {Submessage, TopicLink} from "./types";
+import type {RawReaction} from "./reactions";
```

Comment:
Should this type live in this file instead?

Reply:
I don't see much benefit to moving it here.